src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -73,4 +73,12 @@ public String toString() {
         return '[' + tagName + ']';
     }
 
+    /**
+     * Returns the name of the tag in String
+     * @return
+     */
+    public String getStringTagName() {
```

Comment:
can direcctly call `<A_tag_instance>.tagName`. Because the field is public anyway.